shared/templates/grub2_bootloader_argument/oval.template

```diff
@@ -33,10 +34,14 @@
 {{% set system_with_bios_and_uefi_support = true %}}
 {{%- endif -%}}
 
+{{% if product in ["rhel9", "rhel10"] -%}}
+{{% set bootable_containers_supported = true %}}
+{{%- endif -%}}
```

Comment:
I think this increases technical debt.

Reply:
Yes, so can we go without it? Just have a separate criteria which would check bootc environment + argument existance/non-existance in the `/usr/lib/bootc/kargs.d/*.toml` files?